src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -0,0 +1,139 @@
+  AccordionItem,
+  AccordionItemHeading,
+  AccordionItemPanel,
+  AccordionItemButton
+} from 'src/shared/components/accordion';
+import ExternalReference from 'src/shared/components/external-reference/ExternalReference';
+
+import { getEntityViewerSidebarPayload } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors';
+
+import { RootState } from 'src/store';
+import { EntityViewerSidebarPayload } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarState';
+import { ExternalLink } from 'src/content/app/entity-viewer/types/externalLink';
+
+import styles from './GeneExternalReferences.scss';
+
+type Props = {
+  sidebarPayload: EntityViewerSidebarPayload | null;
+};
+
+const GeneExternalReferences = (props: Props) => {
+  if (!props.sidebarPayload) {
+    return <div>No data to display</div>;
+  }
+  return (
+    <div>
+      <div className={styles.geneDetails}>
+        <div className={styles.geneSymbol}>
+          {props.sidebarPayload.gene.symbol}
+        </div>
+        <div className={styles.stableId}>{props.sidebarPayload.gene.id}</div>
+      </div>
+
+      <div className={styles.sectionHead}>Gene</div>
+      {props.sidebarPayload.gene?.xrefs &&
+        props.sidebarPayload.gene.xrefs.map((xref, key) => {
+          if (xref.links.length === 1) {
+            return (
+              <div key={key}>
+                <ExternalReference
+                  label={xref.links[0].name}
+                  to={xref.links[0].url}
+                  linkText={xref.links[0].value}
+                />
+              </div>
+            );
+          } else {
+            return xref.links[0].name === xref.source_name
+              ? renderXrefGroupWithSameLabels(xref, key)
+              : renderXrefGroupWithDifferentLabels(xref, key);
+          }
+        })}
+
+      {props.sidebarPayload.gene.transcripts && (
+        <div>
+          <div className={styles.sectionHead}>Transcripts</div>
+          {props.sidebarPayload.gene.transcripts.map((transcript, key) => {
+            return (
+              <div key={key} className={styles.transcriptWrapper}>
+                <a href="">{transcript.id}</a>
+                {transcript.xrefs && (
+                  <div className={styles.transcriptXrefs}>
+                    {transcript.xrefs.map((xref, key) => (
+                      <ExternalReference
+                        label={xref.name}
+                        to={xref.url}
+                        linkText={xref.value}
+                        key={key}
+                      />
+                    ))}
+                  </div>
+                )}
+              </div>
+            );
+          })}
+        </div>
+      )}
+    </div>
+  );
+};
+
+const renderXrefGroupWithSameLabels = (xref: ExternalLink, key: number) => {
+  return (
+    <div key={key} className={styles.xrefGroupWithSameLabel}>
+      <span className={styles.xrefGroupSourceName}>{xref.source_name}</span>
+      <span>
```

Comment:
Theoretically, having a div inside a span [is a bad idea](https://stackoverflow.com/a/21400869/3925302).

![image](https://user-images.githubusercontent.com/6834224/78511526-361e2880-7795-11ea-929c-02b2c65dc32d.png)

Reply:
Changed it to span

Reply:
![image](https://user-images.githubusercontent.com/6834224/78564101-ddda3b80-7813-11ea-95f9-55e90ca4d82f.png)

Is this an unintended consequence?

Reply:
No idea how I missed that. Fixed it now.

Reply:
Sorry, but that brings us back to the span containing divs:

![image](https://user-images.githubusercontent.com/6834224/78670729-1c382f00-78d6-11ea-9ed6-724bfe1e3172.png)


Could you either change xrefGroupLinks to div or change the defaultContainer to span with display: block? I think the former will make more sense.

Reply:
Sorry! Changed the container to div.
![Screenshot 2020-04-07 at 13 59 43](https://user-images.githubusercontent.com/3085815/78672233-4985dc80-78d8-11ea-915d-7dc2a10723dd.png)